plugins/confluence-plugin/src/components/PageContent.tsx

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import React, { useState, useEffect } from "react";
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+// import React from "react";
```

Comment:
Let's fix this instead of disabling
```suggestion
import type React from "react";
import { useState, useEffect } from "react";
```